src/dc_lot.rs

```diff
@@ -161,7 +161,7 @@ pub unsafe fn dc_lot_fill(
         }
     }
     (*lot).text2 =
-        dc_msg_get_summarytext_by_raw((*msg).type_0, (*msg).text, (*msg).param, 160i32, context);
+        dc_msg_get_summarytext_by_raw((*msg).type_0, (*msg).text, &mut (*msg).param, 160, context);
```

Comment:
What does the `160` mean here?

Reply:
no idea, I just left it for now, removing unnecessary `i32` suffixes